apps/console/src/features/provisioning/components/outbound-provisioning-connector-delete-wizard.tsx

```diff
@@ -0,0 +1,108 @@
+/**
+ * Copyright (c) 2024, WSO2 LLC. (https://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { IdentifiableComponentInterface } from "@wso2is/core/models";
+import { ConfirmationModal } from "@wso2is/react-components";
+import React, { FunctionComponent, ReactElement } from "react";
+import { Trans, useTranslation } from "react-i18next";
+import { OutboundProvisioningConfigurationInterface } from "../../applications/models/application";
+
+/**
+ * Interface for the outbound provisioning IDP delete confirmation modal props.
+ */
+interface OutboundProvisioningConnectorDeleteWizardPropsInterface extends IdentifiableComponentInterface {
+    isOpen: boolean;
+    onClose: () => void;
+    deletingIdp: OutboundProvisioningConfigurationInterface;
+    onConfirm: (idp: OutboundProvisioningConfigurationInterface) => void;
+}
+
+/**
+ * Outbound provisioning IDP delete confirmation modal component.
+ *
+ * @param props - Props injected to the component.
+ * @returns Outbound provisioning IDP delete confirmation modal component.
+ */
+export const OutboundProvisioningConnectorDeleteWizard: FunctionComponent<
+    OutboundProvisioningConnectorDeleteWizardPropsInterface
+> = (props: OutboundProvisioningConnectorDeleteWizardPropsInterface): ReactElement => {
+
+    const {
+        isOpen,
+        onClose,
+        deletingIdp,
+        onConfirm,
+        [ "data-componentid" ]: testId
+    } = props;
+
+    const { t } = useTranslation();
+
+    return (
+        <ConfirmationModal
+            onClose={ onClose }
+            type="negative"
+            open={ isOpen }
+            assertion={ deletingIdp?.idp }
+            assertionHint={ (
+                <p>
+                    <Trans
+                        i18nKey={
+                            "console:develop.features.applications.confirmations" +
+                            ".deleteOutboundProvisioningIDP.assertionHint"
+                        }
+                        tOptions={ { name: deletingIdp?.idp } }
+                    >
+                        Please type <strong>{ deletingIdp?.idp }</strong> to confirm.
+                    </Trans>
+                </p>
+            ) }
+            assertionType="input"
+            primaryAction="Confirm"
```

Comment:
Shouldn't we i18n the action text?

Reply:
Fixed with 8246d33c5a99b550fe7ac283a1b5f721f2980628